lxd/storage/drivers/driver_zfs_volumes.go

```diff
@@ -2158,6 +2158,23 @@ func (d *zfs) RenameVolume(vol Volume, newVolName string, op *operations.Operati
 
 // MigrateVolume sends a volume for migration.
 func (d *zfs) MigrateVolume(vol Volume, conn io.ReadWriteCloser, volSrcArgs *migration.VolumeSourceArgs, op *operations.Operation) error {
+	if vol.contentType == ContentTypeFS && vol.IsBlockBacked() {
```

Comment:
We only need to do this if volSrcArgs.AllowInconsistent is false. So need a check for it here.

Similar to https://github.com/lxc/lxd/blob/e19b4dfd1f571d5466000b9c8601017dd155280a/lxd/storage/drivers/driver_zfs_volumes.go#L578

Reply:
Right. It's fixed.